jcl/src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -76,6 +82,16 @@ public boolean waitForReferenceProcessing() throws InterruptedException {
 			  }
 			});
 	}
+	/*[ELSE]*/
```

Comment:
This section needs to be commented out in order for this unpre-processed file to compile in tool like Eclipse.

Reply:
fixed